src/tv2-common/actions/executeAction.ts

```diff
@@ -1053,34 +1045,57 @@ async function executeActionCutToCamera<
 		}
 	}
 
-	if (userData.queue || serverInCurrentPart) {
-		await context.core.queuePart(part, [
-			kamPiece,
+	await executePiece(context, settings, kamPiece, userData.queue, part)
```

Comment:
I find it so confusing when we switch languages on things like `kamPiece`, but i know it's just taken from elsewhere.